examples/aggregator/config/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
No idea what's going on here, is it trying to resolve its own location?

Could it be a function defined in `scripts/common`?

Reply:
Maybe... it's apparently the safest bet to get the current folder, but it is not consistent with other places, so I'll change it.

Reply:
It can... but this is a bit different. As we source this, I wanted to get the folder of the script, not of the location where this is called. the method we use in most of the scripts works, as we mostly call them directly, not via `source`.  I think it's a somehow unique usecase. As the comment below, I wouldn't spend too much time on this, as I do think we should remove these files soon and replace them with `.toml`s